packages/react-saasify/src/themes/waves.module.css

```diff
@@ -0,0 +1,598 @@
+ */
+.live-service-demo {
+  background-image: linear-gradient(174deg, #6e60e1 0%, #1b3b87 93%);
+  border-radius: 8px;
+  box-shadow: none !important;
+  filter: drop-shadow(0px 3px 6px #392ab195);
+  overflow: hidden;
+}
+
+.live-service-demo .tabs:before {
+  align-self: center;
+  color: #ffffff90;
+  content: 'Live Demo';
+  font-size: 16px;
+  font-family: 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier,
+    monospace;
+}
+
+.live-service-demo .tabs {
+  background: #313b5250;
+  color: #ffffff80;
+  font-size: 14px;
+  padding-bottom: var(--space-m);
+  padding-top: var(--space-m);
+  padding-left: var(--space-l);
+  padding-right: var(--space-l);
+}
+
+.live-service-demo .tabs .tab:first-of-type {
+  margin-left: auto;
+}
+
+.live-service-demo .tabs .tab:hover {
+  background: transparent;
+  color: #ffffff;
+}
+
+.live-service-demo .tabs .selected-tab {
+  background: transparent;
+  color: #ffffff;
+  font-weight: 600;
+}
+
+.code-block {
+  background: transparent !important;
+  color: white !important;
+  height: auto !important;
+  font-size: 14px;
+  padding: var(--space-m) !important;
+  padding-left: var(--space-l) !important;
+  padding-right: var(--space-l) !important;
+}
+
+/*
+ * BG
+ * ----------------------------
+ */
+
+.section:first-of-type::before {
+  background-image: linear-gradient(104deg, #fcfcfc 13%, #f5f7fb 91%);
+  content: '';
+  position: absolute;
+  height: calc(100% + 100px);
+  min-height: 500px;
+  left: 0;
+  top: 0;
+  width: 100vw;
+}
+
+.section:first-of-type::after {
+  background-image: url("data:image/svg+xml,%3C%3Fxml version='1.0' encoding='UTF-8'%3F%3E%3Csvg width='1440px' height='274px' viewBox='0 0 1440 274' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3C!-- Generator: Sketch 54.1 (76490) - https://sketchapp.com --%3E%3Ctitle%3EArtboard%3C/title%3E%3Cdesc%3ECreated with Sketch.%3C/desc%3E%3Cdefs%3E%3Cpath d='M1440,94.937049 L1440,274 L0,274 L0,68 C163.333342,95.1394992 375.993354,127.635219 637.980036,165.48716 C899.966717,203.339101 1167.30671,179.822397 1440,94.937049 Z' id='path-1'%3E%3C/path%3E%3Cfilter x='-5.7%25' y='-51.7%25' width='111.5%25' height='180.1%25' filterUnits='objectBoundingBox' id='filter-2'%3E%3CfeOffset dx='0' dy='-24' in='SourceAlpha' result='shadowOffsetOuter1'%3E%3C/feOffset%3E%3CfeGaussianBlur stdDeviation='23.5' in='shadowOffsetOuter1' result='shadowBlurOuter1'%3E%3C/feGaussianBlur%3E%3CfeColorMatrix values='0 0 0 0 0.889011549 0 0 0 0 0.889011549 0 0 0 0 0.889011549 0 0 0 0.5 0' type='matrix' in='shadowBlurOuter1'%3E%3C/feColorMatrix%3E%3C/filter%3E%3C/defs%3E%3Cg id='Artboard' stroke='none' stroke-width='1' fill='none' fill-rule='evenodd'%3E%3Cg id='Combined-Shape'%3E%3Cuse fill='black' fill-opacity='1' filter='url(%23filter-2)' xlink:href='%23path-1'%3E%3C/use%3E%3Cuse fill='%23FFFFFF' fill-rule='evenodd' xlink:href='%23path-1'%3E%3C/use%3E%3C/g%3E%3C/g%3E%3C/svg%3E");
```

Comment:
while this works and is fine for now, we should figure out a better way to generalize themes relying on custom assets like this without requiring them to be inline.

Reply:
Yup that would be nice, although this is perfectly workable at the moment.

Reply:
agreed